Basic-Recruit-Training/Person.cpp

```diff
@@ -0,0 +1,69 @@
+#include "Person.h"
+#include <iostream>
+#include <string>
+#include <sstream>
+Person::Person(void)
+{
+    name_ = "";
+    age_ = new int(0);
+}
+
+Person::Person(const std::string& name, int age)
+{
+    name_ = name;
```

Comment:
Just as a tidbit, this can also be done in the constructor like so:

`Person::Person(const std::string& name, int age)`
` :name_(name)`
`{
more code
}`

In general, you'll probably see this one more commonly used since it looks nice and offers some advantages.
 http://stackoverflow.com/questions/926752/why-should-i-prefer-to-use-member-initialization-list